frame/examples/offchain-worker-ping-pong/src/lib.rs

```diff
@@ -0,0 +1,681 @@
+		}
+
+		#[pallet::call_index(2)]
+		#[pallet::weight(0)]
+		pub fn pong_unsigned(
+			origin: OriginFor<T>,
+			_block_number: BlockNumberFor<T>,
+			nonce: u32,
+		) -> DispatchResultWithPostInfo {
+			// This ensures that the function can only be called via unsigned
+			// transaction.
+			ensure_none(origin)?;
+
+			// Emit the PongAckUnauthenticated event
+			Self::deposit_event(Event::PongAckUnauthenticated {
+				nonce,
+				unsigned_type: UnsignedType::RawUnsigned,
+			});
+
+			// now increment the block number at which we expect next unsigned
+			// transaction.
+			let current_block = <system::Pallet<T>>::block_number();
+			<NextUnsignedAt<T>>::put(current_block + T::UnsignedInterval::get());
+			Ok(().into())
+		}
+
+		#[pallet::call_index(3)]
+		#[pallet::weight(0)]
+		pub fn pong_unsigned_with_signed_payload(
+			origin: OriginFor<T>,
+			pong_payload: PongPayload<T::Public, BlockNumberFor<T>>,
+			_signature: T::Signature,
+		) -> DispatchResultWithPostInfo {
+			// This ensures that the function can only be called via unsigned
+			// transaction.
+			ensure_none(origin)?;
+
+			Self::deposit_event(Event::PongAckUnauthenticated {
+				nonce: pong_payload.nonce,
+				unsigned_type: UnsignedType::UnsignedWithSignedPayload,
+			});
+
+			// now increment the block number at which we expect next unsigned
+			// transaction.
+			let current_block = <system::Pallet<T>>::block_number();
+			<NextUnsignedAt<T>>::put(current_block + T::UnsignedInterval::get());
+			Ok(().into())
+		}
+
+		#[pallet::call_index(4)]
+		#[pallet::weight(0)]
+		pub fn add_authority(
+			origin: OriginFor<T>,
+			authority: T::AccountId,
+		) -> DispatchResultWithPostInfo {
+			ensure_root(origin)?;
+
+			ensure!(!Self::is_authority(&authority), Error::<T>::AlreadyAuthority);
+
+			let mut authorities = <Authorities<T>>::get();
+			match authorities.try_push(authority.clone()) {
+				Ok(()) => (),
+				Err(_) => return Err(Error::<T>::TooManyAuthorities.into()),
+			};
+
+			Authorities::<T>::set(authorities);
+
+			Self::deposit_event(Event::AuthorityAdded { authority });
+
+			Ok(().into())
+		}
+
+		#[pallet::call_index(5)]
+		#[pallet::weight(0)]
+		pub fn remove_authority(
+			origin: OriginFor<T>,
+			authority: T::AccountId,
+		) -> DispatchResultWithPostInfo {
+			ensure_root(origin)?;
+
+			ensure!(Self::is_authority(&authority), Error::<T>::NotAuthority);
+
+			let mut authorities = <Authorities<T>>::get();
+			match authorities.iter().position(|a| a == &authority) {
+				Some(index) => authorities.swap_remove(index),
+				None => return Err(Error::<T>::NotAuthority.into()),
+			};
+
+			Authorities::<T>::set(authorities);
+
+			Self::deposit_event(Event::AuthorityAdded { authority });
+
+			Ok(().into())
+		}
+	}
+
+	#[pallet::validate_unsigned]
+	impl<T: Config> ValidateUnsigned for Pallet<T> {
+		type Call = Call<T>;
+
+		/// Validate unsigned calls to this module.
+		///
+		/// By default, unsigned transactions are disallowed, but implementing
```

Comment:
The coding guide mentions 100 line width.

Reply:
Okay just committed 100 line length according the coding guideline:
https://github.com/paritytech/substrate/blob/f81f8ebfdd7c93c39ccf9205b5a91e8496f4b0b2/.editorconfig#L9
If you can confirm that it LGTY.